packages/p2p/stores/buy-sell-store.js

```diff
@@ -0,0 +1,360 @@
+import { observable, action } from 'mobx';
+import { localize } from 'Components/i18next';
+import { height_constants } from 'Utils/height_constants';
+import { requestWS } from 'Utils/websocket';
+import { formatMoney, getDecimalPlaces, getRoundedNumber } from '@deriv/shared';
+import { buy_sell } from '../src/constants/buy-sell';
+import { textValidator, lengthValidator } from 'Utils/validations';
+import { countDecimalPlaces } from 'Utils/string';
+
+export default class BuySellStore {
+    constructor(root_store) {
+        this.root_store = root_store;
+        this.general_store = this.root_store.general_store;
```

Comment:
Try to avoid this pattern. If you're accessing another store try to go through the `root_store` first as this will make it easier to understand the structure of each store as well as make it more maintainable.